internal/__snapshots__/exportTypeScriptSchemas.test.ts.snap

```diff
@@ -187,13 +187,13 @@ import { Time } from "./Time";
 
 /** A single frame of a compressed video bitstream */
 export type CompressedVideo = {
-  /** Timestamp of image */
+  /** Timestamp of video frame */
   timestamp: Time;
 
-  /** Frame of reference for the image. The origin of the frame is the optical center of the camera. +x points to the right in the image, +y points down, and +z points into the plane of the image. */
+  /** Frame of reference for the video. The origin of the frame is the optical center of the camera. +x points to the right in the video, +y points down, and +z points into the plane of the video. */
   frame_id: string;
 
-  /** Compressed video frame data. For packet-based video codecs this data must begin and end on packet boundaries (no partial packets), and must contain enough video packets to decode exactly one image (either a keyframe or delta frame). */
+  /** Compressed video frame data. For packet-based video codecs this data must begin and end on packet boundaries (no partial packets), and must contain enough video packets to decode exactly one image (either a keyframe or delta frame). B frames should be avoided because of their inconsistency with recordings directly written from live sources. */
```

Comment:
B frames are not inconsistent with live sources, live sources just introduce a buffer and some latency to create them. We don't support them because of our source buffering code in studio, not because of some inherent unsuitability for the task. Therefore, i'd modify this to just say "Note: Foxglove Studio does not support display of video streams that include B frames".